.github/workflows/integration-test-flex-suite.yaml

```diff
@@ -1,5 +1,11 @@
 name: Integration test CephFlexSuite
-on: [pull_request]
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
```

Comment:
We also need these same actions for any push to the `release-*` branches

Reply:
added.